neurons/validator/main.py

```diff
@@ -15,8 +15,6 @@
 
     from validator import StableValidator
 
-    sentry_sdk.init(
-        dsn=VALIDATOR_SENTRY_DSN,
-    )
+    sentry_sdk.init(dsn=VALIDATOR_SENTRY_DSN, environment="local")
```

Comment:
`local` should be a config var, otherwise we stop getting notifications from actual Validators